### PR TITLE
Firebase Google Auth

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
@jatinluthra14 I think you should replace _Lead Management_ with _Contact Groups_ :)

**Comment by jatinluthra14** [comment]
@SachinRammoorthy I accidentally pushed lead management changes

**Comment by jatinluthra14** [comment]
@garg-anuj Any changes required?

File: public/signin.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <script src="js/config.js"></script>
+    <script src="js/signin.js"></script>
+	
+    <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/2.5.1/firebaseui.css" />
+  </head>
+  <body>
+	<h1>Welcome to Contact Groups Server</h1>
+    <div id="firebaseui-auth-container"></div>
+  </body>
+</html>
```
Comment (m-murad) [inline public/signin.html]: add a blank line at the end.

File: public/js/signin.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        signInOptions: [
+		{
+			provider: firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+			authMethod: 'https://accounts.google.com'
+		}
+        ]
+      };
+
+var ui = new firebaseui.auth.AuthUI(firebase.auth());
+ui.start('#firebaseui-auth-container', uiConfig);
```
Comment (m-murad) [inline public/js/signin.js]: add a blank line at the end.

File: public/js/load.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    }, function(error) {
+        console.log(error);
+    });
+
+    document.getElementById("sign-out").addEventListener(
+        "click", function() {
+                firebase.auth().signOut();
+        }
+    );
+});
```
Comment (m-murad) [inline public/js/load.js]: add a blank line at the end.

File: public/js/config.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+var config = {
+    apiKey: "",
+    authDomain: "",
+    databaseURL: "",
+    projectId: "",
+    storageBucket: "",
+    messagingSenderId: ""
+  };
+  firebase.initializeApp(config);
```
Comment (m-murad) [inline public/js/config.js]: add a blank line at the end.

File: public/home.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    <script src="https://cdn.firebase.com/libs/firebaseui/2.5.1/firebaseui.js"></script>
+    <script src="js/config.js"></script>
+    <script src="js/load.js"></script>
+</head>
+<body>
+<h1>Hello <div id=username></div></h1>
+<br>
+<button class="btn btn-default" id="sign-out">Sign out</button>
+</body>
+</html>
```
Comment (m-murad) [inline public/home.html]: add a blank line at the end.

**Comment by jatinluthra14** [comment]
@m-murad Done

**Comment by jatinluthra14** [comment]
@m-murad Done

**Pull request closed without merging**